config/locales/en.txt

```diff
@@ -1379,6 +1380,12 @@
   create_naming_title: "Propose Name for Observation #[id]"
   create_new_naming_warn: Sorry, someone else has given this a positive vote, so we had to create a new Naming to accomodate your changes."
 
+  # observations/identify
+  obs_needing_id: Observations Needing Identification
+  obs_needing_id_intro: "Here you can quickly scan unidentified observations, propose a name, or vote on a current name. Use the search bar above to filter location, taxon or user.\n\nIf you feel you can't improve on an identification, \"[:mark_as_reviewed]\" and it won't come up again. (It will still appear for others.)\n\n*\"Lightbox\" shortcuts:*\n[&larr;] [&rarr;] move between observations, [return] propose a name, [/] mark as reviewed."
+  mark_as_reviewed: Mark as reviewed
```

Comment:
Please review the intro I wrote here, it could be better. Please edit directly on this PR!

Also - there's something wrong with my Textile, because it gets cut off, but i can't figure out what.